infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -36,6 +36,13 @@ import (
 
 const setupKeyOutputName = "sa_key"
 
+var (
+	commonRetryableErrors = map[string]string{
```

Comment:
Let's make this public so users can pass it in via option when needed. This will allow us to maintain a common list while other blueprints can extend or pass their own if needed..

Reply:
Map made public